03_myfavorite/src/Components/ResultPage.jsx

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+import Card from './Card';
+import crown from '../img/crown.png';
+
+function ResultPage({ winner }) {
+  return (
+    <Wrapper>
+      <WinnerIs>대망의 우승자는...!</WinnerIs>
+      <CardWrapper>
+        <Card name={winner.name} src={winner.src} />
+        <Crown src={crown} />
+        <ButtonContainer>
+          <RestartButton>다시하기</RestartButton>
+          <ShareButton>공유하기</ShareButton>
+        </ButtonContainer>
+      </CardWrapper>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.div`
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #b09e99;
+`;
+
+const WinnerIs = styled.h1`
+  font-size: 3rem;
+  text-shadow: 2px 2px 4px gray;
+  color: #fee9e1;
+`;
+
+const CardWrapper = styled.div`
+  position: relative;
+  width: 70vh;
+  height: 70vh;
+`;
+
+const Crown = styled.img`
+  position: absolute;
+  width: 20vh;
+  z-index: 1;
+  top: 1vh;
+  left: 10vh;
+`;
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+const RestartButton = styled.button`
+  width: 150px;
+  height: 70px;
+  margin: 10px;
+  border: 1px solid gray;
+  border-radius: 20px;
+  font-size: 1.3rem;
+  background-color: #feb1b7;
+  text-shadow: 2px 2px 4px gray;
+  color: #fee9e1;
+`;
+const ShareButton = styled.button`
+  width: 150px;
+  height: 70px;
+  margin: 10px;
+  border: 1px solid gray;
+  border-radius: 20px;
+  font-size: 1.3rem;
+  text-shadow: 2px 2px 4px gray;
+  color: white;
+  background-color: #a7d676;
+`;
```

Comment:
```suggestion
const ButtonStyle = styled.button`
  width: 150px;
  height: 70px;
  margin: 10px;
  border: 1px solid gray;
  border-radius: 20px;
  font-size: 1.3rem;
  text-shadow: 2px 2px 4px gray;
`;

const RestartButton = styled(ButtonStyle)`
  background-color: #feb1b7;
  color: #fee9e1;
`;

const ShareButton = styled(ButtonStyle)`
  color: white;
  background-color: #a7d676;
`;
```
RestartButton이랑 ShareButton의 css가 중복되는 게 많으니까 이런식으로 작성해주면 어떨까? 🤔